lib/queries.js

```diff
@@ -125,7 +126,7 @@ export function parseFilters(table, filters = {}, params = []) {
       os || browser || device || country
         ? `inner join session on ${table}.session_id = session.session_id`
         : '',
-    pageviewQuery: getFilterQuery('pageview', pageviewFilters, params),
+    pageviewQuery: table === 'event' ? getFilterQuery('event', pageviewFilters, params) : getFilterQuery('pageview', pageviewFilters, params),
```

Comment:
This is confusing. You can just move the `url` param to the `eventFilters`.

Reply:
Thanks for that!

Reply:
Er.. One problem here. This should only be applied to eventFilters, but this function is also used when query other metrics, like session. Maybe we should pass the `table` variable to `getFilterQuery`, but it will require further refactoring of the code